### PR TITLE
Create CoA-premium-pack-readme.md

### PR DESCRIPTION
**Pull request by mayagoldb**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by glicht** [comment]
@mayagoldb build is failing.

you need to add a front matter (header). You can use the following at the start of the doc:
```yml
---
id: courses-of-action
title: Courses of Action
description: This Courses of Action Pack contains intelligence-driven Courses of Action (COA) defined by Palo Alto Networks Unit 42 team that will enable you to handle MITRE ATT&CK techniques and sub-techniques in an organised and automated manner.
---
```

File: content-repo/extra-docs/articles/CoA-premium-pack-readme.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Possible indicator types to store MITRE ATT&CK techniques: STIX Report, STIX Malware, etc.
+*** This method requires a Cortex XSOAR TIM license.	
+From incident - For incident types that include techniques in the incident layout, the incident handling playbook can include the “MITRE ATT&CK - Courses of Action'' wrapper playbook as a sub-playbook. In this case, all of the associated techniques will be remediated by CoAs in addition to the specific IOCs.
+Possible incidents: Cortex XDR incidents and alerts.
+Specific technique trigger - Remediate specific techniques using their corresponding COA playbook.
+Example - techniques fetched from the MITRE ATT&CK feed.
+Job - Use a job to trigger the “MITRE ATT&CK - Courses of Action” wrapper playbook to keep the environment up to date with all relevant CoAs.
+Pack Configurations
+To configure and use the playbooks in this pack, you should do the following:
+
```
Comment (michalgold) [inline content-repo/extra-docs/articles/CoA-premium-pack-readme.md:37]: Should reference the dependencies and their documentation. 

Additions for the Unit42 feed- 
*reference playbook viewer, add image as well. 
*Explain that the reports are indicators of type STIX report
*Explain that the Mitre Techniques are referenced as feed related indicators under stix report
*Each Mitre Technique is clickable, quick view will show. For more details click full details to read additional information from Mitre.  

Additions for Mitre Att&ck:
*Mention the Mitre dashboard to show the status across incidents
*Mention that without configuring this pack there will not be any data populated on the indicator (even after we create it via unit42)

Additions for PAN-OS:
*List all the license pre-requisites and explain they are needed for certain actions. 
*Highlight the approval flow for policy changes
*Explain FW vs Panorama

@mayagoldb @kirbles19 @yaron-libman

File: content-repo/extra-docs/articles/CoA-premium-pack-readme.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+*** This method requires a Cortex XSOAR TIM license.	
+From incident - For incident types that include techniques in the incident layout, the incident handling playbook can include the “MITRE ATT&CK - Courses of Action'' wrapper playbook as a sub-playbook. In this case, all of the associated techniques will be remediated by CoAs in addition to the specific IOCs.
+Possible incidents: Cortex XDR incidents and alerts.
+Specific technique trigger - Remediate specific techniques using their corresponding COA playbook.
+Example - techniques fetched from the MITRE ATT&CK feed.
+Job - Use a job to trigger the “MITRE ATT&CK - Courses of Action” wrapper playbook to keep the environment up to date with all relevant CoAs.
+Pack Configurations
+To configure and use the playbooks in this pack, you should do the following:
+ 
+# Playbook Triggers and configuration:
```
Comment (mayagoldb) [inline content-repo/extra-docs/articles/CoA-premium-pack-readme.md:38]: Trigger definitions:
1) Investigate specific techniques using manual trigger:
Manually create an incident with type "MITRE ATT&CK CoA

2) Embedded within existing incident workflow -  Sub-Playbook

3) Scheduled Job - Compliance

4) Intelligence Feed triggered Job - Act upon threat reports, TIM License recommended

File: content-repo/extra-docs/articles/CoA-premium-pack-readme.md (hunk 1 above)
Comment (mayagoldb) [reply content-repo/extra-docs/articles/CoA-premium-pack-readme.md:37]: Unit 42 part should also be emphasized in the Unit 42 Pack documentation and link to this pack.

**Comment by yaron-libman** [comment]
approved

**Review by yaron-libman — APPROVED** [review]

**Pull request merged**